docker/android/Makefile

```diff
@@ -0,0 +1,57 @@
+GIT_COMMIT = $(shell git rev-parse --short HEAD)
+
+# Software Versions, URLs, and Checksums
+NVM_VERSION = v0.33.11
+NVM_URL = https://raw.githubusercontent.com/creationix/nvm/$(NVM_VERSION)/install.sh
+NVM_INSTALL_SCRIPT = nvm_$(NVM_VERSION)_install.sh
+NVM_NODE_VERSION = v10.14.1
```

Comment:
We're not using the same Node version as we're installing in `installer.sh` (v10.14.0), we should pick one and stick with it.

Reply:
Good point. Will change that, thanks!